### PR TITLE
Add genai docs

### PR DESCRIPTION
**Pull request by natke**
Initial draft of GenAI docs: staged here: https://natke.github.io/onnxruntime/docs/genai/

### CODE REVIEW

File: docs/genai/tutorials/phi2-python.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+```bash
+python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 -o ./example-models/phi2-int4-cpu
+```
+You can replace the name of the output folder specified with the `-o` option with a folder of your choice.
+
+After you run the script, you will see a series of files generated in this folder. They include the HuggingFace configs for your reference, as well as the following generated files used by ONNX Runtime GenAI.
+
+`model.onnx`: the phi-2 ONNX model
+`model.onnx.data`: the phi-2 ONNX model weights
```
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi2-python.md:37]: These don't appear as a list in the staged docs

Comment (natke) [reply docs/genai/tutorials/phi2-python.md]: Fixed

File: docs/genai/tutorials/phi2-python.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+# Language generation in Python with phi-2
+
+## Setup and installation
+
+Install the ONNX Runtime GenAI Python package using the [installation instructions](../howto/install.md).
+
+## Build phi-2 ONNX model
+
+The onnxruntime-genai package contains a model builder that generates the phi-2 ONNX model using the weights and config on Huggingface. The tools also allows you to load locally stored weights, or convert from GGUF format. For more details, see [how to build models](../howto/build-models.md)
```
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi2-python.md:18]:
```suggestion
The onnxruntime-genai package contains a model builder that generates the phi-2 ONNX model using the weights and config on Huggingface. The tools also allows you to download the weights from Hugging Face, load locally stored weights, or convert from GGUF format. For more details, see [how to build models](../howto/build-models.md)
```

File: docs/genai/tutorials/phi2-python.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+If using the `-m` option shown here, which downloads from HuggingFace, you will need to login into HuggingFace.
```
Comment (kunal-vaishnavi) [inline docs/genai/tutorials/phi2-python.md:20]:
```suggestion
If using the `-m` option shown here, you will need to login into Hugging Face.
```

**Review by baijumeswani — APPROVED** [review]

**Review by baijumeswani — APPROVED** [review]

**Review by yufenglee — APPROVED** [review]

**Review by kunal-vaishnavi — APPROVED** [review]

**Pull request merged**